README.md

```diff
@@ -1,3 +1,7 @@
-New repository for binaries:
+Building and waiting on PRs to complete on the fork takes too long.
+
+My repository for binaries:
 
 https://github.com/FullofQuarks/Windows-Ledger-Binaries
```

Comment:
This repository (https://github.com/FullofQuarks/Windows-Ledger-Binaries) is not mine. I've updated README to include a direct link for anyone who might be interested.